README.MD

```diff
@@ -0,0 +1,76 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> One paragraph statement about the project.
+
+![screenshot](./app_screenshot.png)
+
+Additional description about the project and its features.
+
+## Built With
+
+- Major languages
+- Frameworks
+- Technologies used
+
+## Live Demo
+
+[Live Demo Link](https://mehdi-rh.github.io/Blog-Capstone/)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
```

Comment:
- [OPTIONAL] Kindly, Add a descriptive README file to your project, it will be great to add instructions to set the project up locally and instructions on  how to run the project. This includes `npm install` , `npm start` etc